src/VecSim/spaces/IP/IP_AVX512.cpp

```diff
@@ -24,20 +23,63 @@ float InnerProductSIMD16Ext_AVX512_impl(const void *pVect1v, const void *pVect2v
         __m512 v2 = _mm512_loadu_ps(pVect2);
         pVect2 += 16;
         sum512 = _mm512_add_ps(sum512, _mm512_mul_ps(v1, v2));
+        // sum512 = _mm512_fmadd_ps(v1, v2, sum512);
```

Comment:
remove?

Reply:
I left it as a reminder that its exists, maybe it will be useful someday.
The compiler should be able to compile both lines into a single `fmadd_ps` command for the CPU